### PR TITLE
[TACHYON-1488] Improve javadoc in AbstractLocalTachyonCluster

### PR DESCRIPTION
**Pull request by caadecarvalho**

### CODE REVIEW

**Comment by yupeng9** [comment]
Jenkins, add to whitelist

**Comment by yupeng9** [comment]
Jenkins, okay to test

**Comment by yupeng9** [comment]
Jenkins, test this please

**Comment by yupeng9** [comment]
LGTM. Thanks for the doc improvement!

File: minicluster/src/main/java/tachyon/master/AbstractLocalTachyonCluster.java [hunk 1]
```diff
@@ -422,26 +422,26 @@ public void run() {
...
-   * Get master's {@link tachyon.conf.TachyonConf}.
+   * Gets master's {@link tachyon.conf.TachyonConf}.
    */
   public TachyonConf getMasterTachyonConf() {
     return mMasterConf;
   }
 
   /**
-   * Reset context when stop the local tachyon cluster.
+   * Resets context when stops the local tachyon cluster.
```
Comment (aaudiber) [inline minicluster/src/main/java/tachyon/master/AbstractLocalTachyonCluster.java]: `Resets contexts. This is called when the cluster is stopped.`

File: minicluster/src/main/java/tachyon/master/AbstractLocalTachyonCluster.java [hunk 2]
```diff
@@ -354,11 +354,11 @@ public TachyonConf newTestConf() throws IOException {
     testConf.set(Constants.WORKER_DATA_BIND_HOST, mHostname);
     testConf.set(Constants.WORKER_WEB_BIND_HOST, mHostname);
 
-    // Perform immediate shutdown of data server. Graceful shutdown is unnecessary and slow
+    // Performs an immediate shutdown of data server. Graceful shutdown is unnecessary and slow
     testConf.set(Constants.WORKER_NETWORK_NETTY_SHUTDOWN_QUIET_PERIOD, Integer.toString(0));
     testConf.set(Constants.WORKER_NETWORK_NETTY_SHUTDOWN_TIMEOUT, Integer.toString(0));
 
-    // Setup tiered store
+    // Setups tiered store
```
Comment (aaudiber) [inline minicluster/src/main/java/tachyon/master/AbstractLocalTachyonCluster.java]: `Sets up the tiered store`

File: minicluster/src/main/java/tachyon/master/AbstractLocalTachyonCluster.java [hunk 3]
```diff
@@ -279,14 +279,14 @@ public void stop() throws Exception {
...
-   * Stop the tachyon filesystem's service thread only.
+   * Stops the tachyon filesystem's service thread only.
    *
    * @throws Exception when the operation fails
    */
   public abstract void stopTFS() throws Exception;
 
   /**
-   * Cleanup the underfs cluster test folder only.
+   * Cleanups the underfs cluster test folder only.
```
Comment (aaudiber) [inline minicluster/src/main/java/tachyon/master/AbstractLocalTachyonCluster.java]: Cleans up

**Comment by aaudiber** [comment]
Thanks for the doc improvement! I've left a few comments

**Comment by aaudiber** [comment]
ok to test

**Comment by gpang** [comment]
This looks good after the suggested minor changes. Thanks!

**Comment by aaudiber** [comment]
LGTM

**Pull request merged**